### DISCUSSION TITLE
Ability to select a block as "ground level" on a page

### DISCUSSION THREAD

**Discussion by Limezy** [athensresearch/athens#902] [Feature Requests]
Hi,

I'll try to explain the best I can my idea. 

The thing is outliners are incredibly helpful tools, but in the end not all documents are written using purely bullet points. 
Usual documents are written using a hierarchy of titles (Header 1, Header 2, Header 3...), then a "ground" level made of paragraphs and sentences, and then some "underground" levels made of bullet or number lists / sublists.

I think it would be wonderful to be able to select one of the blocks of a page and define it as ground level. Then, all higher level bullets would be considered by the software as headers, and all under levels would be considered by the software as bullet lists / sublists. 

That would allow both for 
* A simplified Markdown export of each page
* A great "Document view" where bullets would "fade" and each level be rendered according to its level in real time, just like the Typora markdown editing software for instance. That would be, IMHO, far superior from the Roam Research "Document view" that basically just removes the bullets.

I'll try very soon to make some mockups and share them here for better understanding

**Comment by Limezy** [comment]
I've tried to make mockups to explain better : 
### This is a test page in Athens, standard view, where a bullet could be defined as "ground level"

<img width="1552" alt="Screenshot 2021-04-02 at 20 51 04" src="https://user-images.githubusercontent.com/24638389/113424123-30880c80-9402-11eb-9a71-c1cd536d212e.png">

### This is how that same page would look when exported to .md
![image](https://user-images.githubusercontent.com/24638389/113424196-4bf31780-9402-11eb-8544-9058b2a641b5.png)

### This is how that same page would look in "document mode"
<img width="1552" alt="Screenshot 2021-04-02 at 20 51 04 copy" src="https://user-images.githubusercontent.com/24638389/113424411-b015db80-9402-11eb-895a-170529006ded.png">

As you see, there is some natural way of defining "higher than ground" bullets as H3, H2, H1. "ground" bullets as sentences, or paragraphs, and "lower than ground" bullets as lists and sub-lists.

**Reply by jefftangx** [reply]
I love this idea

**Reply by shanberg** [reply]
I wonder how you'd communicate this feature to users who don't already have this model in mind.

**Reply by Limezy** [reply]
> I wonder how you'd communicate this feature to users who don't already have this model in mind.

I think that if the text edition is "organic" and natural enough, could work. I've given a few ideas in the [dedicated discussion](https://github.com/athensresearch/athens/discussions/910#discussioncomment-588651)
